test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -490,7 +493,17 @@ def run(self, test):
         self.result = result = self._makeResult()
         self.result.start = datetime.datetime.now()
         try:
-            test(result)
+            for i in range(self.repetition):
+                self.logger.info("====================================")
+                self.logger.info("tests are running (" + str(i + 1) + ") time")
```

Comment:
`self.logger.info("Running tests %d time" % i + 1)`

Reply:
I don't think you need both this message and the below one.  We only need to be told once that the tests were repeated N times.

Reply:
Agree with @bhroam.